underfs/s3a/src/main/java/alluxio/underfs/s3a/S3AUnderFileSystem.java

```diff
@@ -79,6 +79,9 @@
   /** Threshold to do multipart copy. */
   private static final long MULTIPART_COPY_THRESHOLD = 100 * Constants.MB;
 
+  /** Minimum size of each copy part. */
+  private static final long MULTIPART_COPY_SIZE = 5 * Constants.MB;
```

Comment:
This is the minimum part size and the same size we use for uploads. The default multipart copy threshold was originally 5GB (which reduced to 100MB), so it makes sense to reduce the minimum part size if we plan to use multipart copy for most files.